docs/docs/dev/languages.md

```diff
@@ -13,3 +13,47 @@ To start up the system with your chosen language, use the following command:
    ```
    
 Note that not every machine learning model is compatible with every language. For model-language compatibility, see [here](model_training.md#model-policies).
+
+
+### Adding support for a new language
```

Comment:
```suggestion
## Adding support for a new language
```

Lowering header level, to avoid sphinx complaining about `WARNING: Non-consecutive header level increase; H1 to H3`